docker/entrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# Define the start command
+START_COMMAND="/usr/local/bin/sim-cli"
+
+# Check for a custom configuration path
+if [[ ! -z ${CONFIG_PATH} ]]; then
+    START_COMMAND="$START_COMMAND --config $CONFIG_PATH"
```

Comment:
Do we need to parse each config option? 

Looking [here](https://refine.dev/blog/docker-entrypoint/#difference-between-entrypoint-and-cmd): "Whatever you provide through entrypoint actually appends to the existing entrypoint instruction in the dockerfile", we should be able to just specify any CLI flags and they'll be passed straight through if we make our entrypoint `sim-cli`.